src/esn.calendar.libs/i18n/en.json

```diff
@@ -243,6 +243,7 @@
   "Submitting importing calendar request": "Submitting importing calendar request...",
   "Successfully subscribed to calendar": "Successfully subscribed to calendar",
   "Successfully subscribed to calendars": "Successfully subscribed to calendars",
+  "Successfully added shared calendar": "Successfully added shared calendar",
```

Comment:
```suggestion
  "Successfully added shared calendar": "shared calendar added successfully",
```

Reply:
humm , yes more logic i am just keeping the some message in code :p

Reply:
Why? "Successfully added shared calendar" sounds more intuitive to me.

Reply:
Though, if we really want to be grammatically correct here, it should be "Successfully added the shared calendar".

Reply:
because it is missing something between the `added` and `shared`.

could be `Successfully added the shared calendar` or `Successfully added a shared calendar`.

`Shared calendar added successfully` sounds more minimalistic